.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Description
+<!-- Describe what problem this is solving, and how it's solved.-->
+
+## Testing 
+<!-- Explain how this change is tested. -->
```

Comment:
mention a TODO list before merging can be considered.

Also put a disclaimer that any and all contributions will be released under Apache 2.0 and the author agrees to that by opening the pull request.